src/Tools/dotnet-monitor/Extensibility/ProgramExtension.cs

```diff
@@ -203,7 +205,7 @@ private void ValidateExecutable(IFileInfo entrypointInfo)
         {
             if (!entrypointInfo.Exists || entrypointInfo.IsDirectory || entrypointInfo.PhysicalPath == null)
             {
-                _logger.ExtensionProgramMissing(_extensionName, Path.Combine(_targetFolder, _exePath), Declaration.Program);
+                _logger.ExtensionProgramMissing(_extensionName, Path.Combine(_targetFolder, _exePath), Declaration.ExecutableFileName);
```

Comment:
In theory, this isn't specified when we validate the AssemblyFileName, correct? Do we just want to pass the file name as a parameter so we don't have to do the if/else check again to determine which one to use?

Reply:
You're absolutely right... but if you are okay with it, I'd rather defer fixing this to a separate change to cleanup the entire ProgramExtension class. It's kind of a mess right now and several of the fields don't represent what their names suggest. I'd like to take a separate pass over it after this PR is merged in.